### DISCUSSION TITLE
Is there a way to see discharge due to snowmelt processes in wflow?

### DISCUSSION THREAD

**Discussion by abdulrehman565** [Deltares/Wflow.jl#285] [Q&A] [answered]
I was doing modelling on a certain region with glaciers, I was wondering if I could see the flow due to glacier melt that is contributing at a certain point. Is there a way to print out only the flows due to glacier melt?

**Comment by verseve** [comment, marked as answer]
Wflow does not track water sources, so it is not possible to see the contribution of glacier melt to river flow.

**Comment by abdulrehman565** [comment]
but does wflow add the effect of glaciers and snow in the resulting flows?

**Comment by verseve** [comment]
Yes, melt is added to infiltration into the soil.

**Comment by markhegnauer** [comment]
Although not possible at this time, a tracking of water sources would be a nice added feature for wflow.
